docs/data/base/components/modal/NestedModal.tsx

```diff
@@ -109,7 +109,8 @@ const style = (theme: Theme) => ({
   left: '50%',
   transform: 'translate(-50%, -50%)',
   width: 400,
-  backgroundColor: theme.palette.mode === 'dark' ? '#0A1929' : 'white',
-  border: '2px solid currentColor',
+  borderRadius: '12px',
   padding: '16px 32px 24px 32px',
+  backgroundColor: theme.palette.mode === 'dark' ? '#0A1929' : 'white',
+  boxShadow: 16,
```

Comment:
Let's use a literal value for `box-shadow`. Having a theme-related value does work well on the docs site, but fails when the demo is opened in codesandbox (there's a different theme provider there).

(+ other similar cases)